js-extensions/spec/ExtensionStore-spec.js

```diff
@@ -1,7 +1,9 @@
 var jsTest = require('@jenkins-cd/js-test');
 var expect = require('chai').expect;
-var ExtensionStore = require('../dist/ExtensionStore').ExtensionStore;
-var ClassMetadataStore = require('../dist/ClassMetadataStore').instance;
+var ExtensionStore = require('../dist/ExtensionStore').default;
+var ClassMetadataStore = require('../dist/ClassMetadataStore').default;
+var resourceLoadTracker = new (require('../dist/ResourceLoadTracker').default)();
```

Comment:
Ah there you go ... so do you really need the test related exports in the top level API? See earlier comment.